.travis.yml

```diff
@@ -1,34 +1,41 @@
 ---
```

Comment:
all these changes will be overwritten by our modulesync_config:
https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L2-L23

Which results in this travis file:
https://github.com/voxpupuli/puppet-tea/blob/master/.travis.yml

Our current template allows a few customizations, let me know if any of the underlying changes are still needed.